### PR TITLE
LPS-151734 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related Issue: https://issues.liferay.com/browse/LPS-151734

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @marcelabc, can you please check if these failures are related to your changes? Thanks

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/data/provider/RelatedModelsFDSDataProvider.java [hunk 1]
```diff
@@ -78,14 +82,47 @@ public List<RelatedModel> getItems(
...
+					objectEntryId, objectDefinition, objectRelationship),
+				entryMap -> {
+					Long titleFieldId =
+						objectDefinition.getTitleObjectFieldId();
+
+					String titleFieldName =
+						objectDefinition.getPKObjectFieldDBColumnName();
+
+					if (titleFieldId > 0) {
+						ObjectField titleField =
```
Comment (mateussgn) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/data/provider/RelatedModelsFDSDataProvider.java:99]: Would it not be better to rename this variable to objectfField instead?

**Review by mateussgn — APPROVED** [review]

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/data/provider/RelatedModelsFDSDataProvider.java (hunk 1 above)
Comment (p-albuquerque) [reply modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/frontend/data/set/data/provider/RelatedModelsFDSDataProvider.java:99]: You are right, I will fix this

**Pull request closed without merging**